app/assets/stylesheets/application.scss

```diff
@@ -160,4 +160,13 @@ select.selectpicker + .dropdown-toggle::after {
 
 .percentage-selector {
   width: 110px;
-}
\ No newline at end of file
+}
+
+.form-yesno > div > div {
+	width: 3rem;
+	text-align: center;
+}
+
+.form-yesno > div > div > label {
```

Comment:
These kinds of rules are pretty brittle. Can we add some classes or IDs here?

Reply:
I was able to rewrite these rules to use classes.